overlord/fdestate/fdemgr.go

```diff
@@ -599,6 +599,22 @@ func (m *FDEManager) CheckRecoveryKey(rkey keys.RecoveryKey, containerRoles []st
 	return nil
 }
 
+// systemEncrypted reports whether FDE is enabled on the system.
+// It returns true if FDE is enabled, or false otherwise.
+func (m *FDEManager) systemEncrypted() (bool, error) {
+	var s FdeState
+	err := m.state.Get(fdeStateKey, &s)
+	if err != nil {
```

Comment:
Do you want to look for `ErrNoState`?
```suggestion
	 if err != nil && !errors.Is(err, state.ErrNoState) {
```